java-client/session/src/main/java/io/deephaven/client/impl/DataAndExports.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.client.impl;
+
+import com.google.protobuf.ByteString;
+import io.deephaven.client.impl.ObjectService.MessageStream;
+import io.deephaven.client.impl.ObjectService.Bidirectional;
+import io.deephaven.client.impl.ObjectService.Fetchable;
+import io.deephaven.proto.backplane.grpc.Data;
+import io.deephaven.proto.backplane.grpc.FetchObjectResponse;
+
+import java.io.Closeable;
+import java.nio.ByteBuffer;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+
+/**
+ * Data and exports are sent from the server to the client as part of a {@link Fetchable#fetch() fetch} or
+ * {@link Bidirectional#messageStream(MessageStream) bidirectional message stream}. The client is responsible for
+ * managing the {@link #exports()}.
```

Comment:
What does "managing" mean? Seems like "keep track of and close when no longer needed".